sol/dmap.sol

```diff
@@ -46,23 +51,5 @@ contract Dmap {
             mstore(0, SIG_LOCK)
             revert(0, 4)
         }
-    }
-
-    function slot(bytes32 s) external view
-      returns (bytes32) {
-        assembly {
-            mstore(0, sload(s))
-            return(0, 32)
-        }
-    }
-
-    function pair(bytes32 s) external view
-      returns (bytes32 meta, bytes32 data) {
-        assembly {
-            mstore(0, sload(s))
-            mstore(32, sload(add(s, 1)))
-            return(0, 64)
-        }
-    }
-
+    }}
```

Comment:
should revert if neither case matches

Reply:
not sure how I feel about ignoring the sig bytes, it's a fun thing and saves gas but we should consider downsides fully before choosing